oak/server/oak_node.cc

```diff
@@ -359,4 +369,25 @@ wabt::interp::HostFunc::Callback OakNode::OakWaitOnChannels(wabt::interp::Enviro
   };
 }
 
+wabt::interp::HostFunc::Callback OakNode::OakChannelFind(wabt::interp::Environment* env) {
+  return [this, env](const wabt::interp::HostFunc* func, const wabt::interp::FuncSignature* sig,
+                     const wabt::interp::TypedValues& args, wabt::interp::TypedValues& results) {
+    LogHostFunctionCall(func, args);
+
+    uint32_t offset = args[0].get_i32();
+    uint32_t length = args[1].get_i32();
+    results[0].set_i64(0);
+
+    auto base = env->GetMemory(0)->data.begin() + offset;
+    std::string port_name(base, base + length);
```

Comment:
Do we need to check boundaries here? AFAICT this may well be past the end of the memory.

Reply:
Pre-existing so added #219